examples/tstool.py

```diff
@@ -0,0 +1,623 @@
+                STATE       = 'STATE',
+                DSTATE      = 'Desktop',
+                CONNTIME    = 'ConnectTime',
+                DISCTIME    = 'DisconnectTime',
+            )
+        
+        header2 = template.replace(' <','=<').format(
+                SESSIONNAME = '',
+                USERNAME    = '',
+                ID          = '',
+                STATE       = '',
+                DSTATE      = '',
+                CONNTIME    = '',
+                DISCTIME    = '',
+            )
+
+        header_verbose = ''
+        header2_verbose = ''
+        if options.verbose:
+            header_verbose = template_verbose.format(
+                                  CLIENTNAME = 'ClientName',
+                                  REMOTEIP = 'RemoteAddress',
+                                  RESOLUTION = 'Resolution',
+                                  TIMEZONE = 'ClientTimeZone'
+                              )
+            header2_verbose = template_verbose.replace(' <','=<').format(
+                                  CLIENTNAME = '',
+                                  REMOTEIP = '',
+                                  RESOLUTION = '',
+                                  TIMEZONE = ''
+                              )
+        result.append(header+header_verbose)
+        result.append(header2+header2_verbose+'\n')
+        
+        for i in self.sessions:
+            connectTime = self.sessions[i]['ConnectTime']
+            connectTime = connectTime.strftime(r'%Y/%m/%d %H:%M:%S') if connectTime.year > 1601 else 'None'
+
+            disconnectTime = self.sessions[i]['DisconnectTime']
+            disconnectTime = disconnectTime.strftime(r'%Y/%m/%d %H:%M:%S') if disconnectTime.year > 1601 else 'None'
+            userName = self.sessions[i]['Domain'] + '\\' + self.sessions[i]['Username'] if len(self.sessions[i]['Username']) else ''
+
+            row = template.format(
+                SESSIONNAME = self.sessions[i]['SessionName'],
+                USERNAME    = userName,
+                ID          = i,
+                STATE       = self.sessions[i]['state'],
+                DSTATE      = desktop_states[self.sessions[i]['flags']],
+                CONNTIME    = connectTime,
+                DISCTIME    = disconnectTime,
+            )
+            row_verbose = ''
+            if options.verbose:
+                row_verbose = template_verbose.format(
+                                    CLIENTNAME = self.sessions[i]['ClientName'],
+                                    REMOTEIP = self.sessions[i]['RemoteIp'],
+                                    RESOLUTION = self.sessions[i]['Resolution'],
+                                    TIMEZONE = self.sessions[i]['ClientTimeZone']
+                                )                
+            result.append(row+row_verbose)
+
+        for row in result:
+            print(row)
+
+    def do_tasklist(self):
+        options = self.__options
+        if not options.verbose:
```

Comment:
There's a good amount of duplicated code in the two branches, I think it would be better to make the calls and check the verbose condition after just to decide what to print.